.eslintrc.json

```diff
@@ -16,6 +21,13 @@
     "rules": {
         "no-debugger": "off",
         "no-console": 0,
-        "class-methods-use-this": "off"
+        "class-methods-use-this": "off",
+        "@typescript-eslint/no-explicit-any": 2,
+        "prettier/prettier": [
+            "error",
+            {
+                "endOfLine": "auto"
+            }
+          ]
```

Comment:
I tried to run npm run lint, but it fall with errors.
Could you please check if it runs successfully on your local machine?